unittests/PhasarLLVM/DataFlow/IfdsIde/Problems/IDELinearConstantAnalysisSwiftTest.cpp

```diff
@@ -20,32 +22,32 @@ class IDELinearConstantAnalysisSwiftTest : public ::testing::Test {
 protected:
   static constexpr auto PathToSwiftTestFiles = PHASAR_BUILD_SWIFT_SUBFOLDER(
       "linear_constant/"); // Function - Line Nr - Variable - Value
+  const std::vector<std::string> EntryPoints = {"main"};
+
+  // Function - Line Nr - Variable - Value
   using LCACompactResult_t = std::tuple<std::string, std::size_t, std::string,
                                         IDELinearConstantAnalysisDomain::l_t>;
-  std::unique_ptr<LLVMProjectIRDB> IRDB;
 
-  void SetUp() override {}
+  void SetUp() override { ValueAnnotationPass::resetValueID(); }
```

Comment:
```suggestion
```

This is not needed anymore. The LLVMProjectIRDB's ctors do not use the ValueAnnotationPass anymore